src/main/resources/db/migration/V3__create_post_table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE SEQUENCE post_id_seq START 1 INCREMENT BY 50;
+
+CREATE TABLE POSTS
+(
+    id                BIGINT                                                        DEFAULT nextval('post_id_seq') NOT NULL,
+    title             VARCHAR(255) UNIQUE                                  NOT NULL,
+    slug              VARCHAR(255) UNIQUE                                  NOT NULL,
+    short_description VARCHAR(255) UNIQUE                                  NOT NULL,
```

Comment:
short_description need not be a unique column.

Reply:
solved